plugins/woocommerce/src/Internal/Features/FeaturesController.php

```diff
@@ -90,32 +90,37 @@ class FeaturesController {
 	 */
 	public function __construct() {
 		$features = array(
-			'analytics'              => array(
+			'analytics'                     => array(
 				'name'               => __( 'Analytics', 'woocommerce' ),
 				'description'        => __( 'Enables WooCommerce Analytics', 'woocommerce' ),
 				'is_experimental'    => false,
 				'enabled_by_default' => true,
 				'disable_ui'         => false,
 			),
-			'new_navigation'         => array(
+			'new_navigation'                => array(
 				'name'            => __( 'Navigation', 'woocommerce' ),
 				'description'     => __( 'Adds the new WooCommerce navigation experience to the dashboard', 'woocommerce' ),
 				'is_experimental' => false,
 				'disable_ui'      => false,
 			),
-			'new_product_management' => array(
+			'new_product_management'        => array(
```

Comment:
I think we could remove this `feature` now, given it was disabled anyway and just replace it with the one below.

Reply:
As mentioned above, on second thought I think it would be better to keep it for at-least one release.